ops/handshake/message.go

```diff
@@ -1,52 +1,35 @@
 package handshake
 
+import "encoding/binary"
+
+type messageID int8
+
 const (
-	keepAlive = "keep-alive"
-	choke = "choke"
-	unchoke = "unchoke"
-	interested = "interested"
-	notIntersted = "not-interested"
-	have = "have"
-	request = "request"
-	cancel = "cancel"
-	piece = "piece"
+	keepAlive    messageID = 0
+	choke        messageID = 0
+	unchoke      messageID = 1
+	interested   messageID = 2
+	notIntersted messageID = 3
+	have         messageID = 4
+	bitField     messageID = 5
+	request      messageID = 6
+	piece        messageID = 7
+	cancel       messageID = 8
 )
 
-type attrib struct {
-	prefix []byte
-	messageID byte
-	payload []byte
+type Message struct {
+	MsgID   uint8
```

Comment:
The type of MsgID should be messageID, don't you think ?

Reply:
Got it. Will do so.